docs/src/concepts.md

```diff
@@ -0,0 +1,123 @@
+# Concepts
+
+This package enhances existing concepts from Sublime Text and introduces new concepts not native to Sublime Text. This page provides an overview of the most important capabilities. The capabilities/concepts are accessible in different ways. Some are accessible via the Context Menu by right-clicking with your mouse, or via the top Menu Bar. Others are part of an existing workflow. Almost all capabilities can also be bound to a key with a key binding.
+
+## Auto Complete [cooperating]
```

Comment:
I wonder if those words in the square brackets are necessary or even that it's clear what those mean.